Sources/AWSLambdaRuntimeCore/Documentation.docc/Deployment.md

```diff
@@ -0,0 +1,602 @@
+- Deleting S3 object with key APIGatewayLambda/1b5a27c048549382462bd8ea589f7cfe           
+- Deleting S3 object with key APIGatewayLambda/396d2c434ecc24aaddb670bd5cca5fe8.template  
+- Deleting Cloudformation stack APIGatewayLambda
+
+Deleted successfully
+```
+
+## Deploy your Lambda function with the AWS Cloud Development Kit (CDK)
+
+The AWS Cloud Development Kit is an open-source software development framework to define cloud infrastructure in code and provision it through AWS CloudFormation. The CDK provides high-level constructs that preconfigure AWS resources with best practices, and you can use familiar programming languages like TypeScript, Javascript, Python, Java, C#, and Go to define your infrastructure.
+
+To use the CDK, you need to [install the CDK CLI](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html) on your machine. The CDK CLI provides a set of commands to manage your CDK projects.
+
+Use the CDK when you want to define your infrastructure in code and manage the deployment of your Lambda function and other AWS services.
+
+### Create a CDK project
+
+To create a new CDK project, use the `cdk init` command. The command creates a new directory with the project structure and the necessary files to define your infrastructure.
+
+```sh
+# In your Swift Lambda project folder
+mkdir infra && cd infra
+cdk init app --language typescript
+```
+
+In this example, the code to create a Swift Lambda function with the CDK is written in TypeScript. The following code creates a new Lambda function with the `swift` runtime.
+
+It requires the `@aws-cdk/aws-lambda` package to define the Lambda function. You can install the dependency with the following command:
+
+```sh
+npm install aws-cdk-lib constructs
+```
+
+Then, in the lib folder, create a new file named `swift-lambda-stack.ts` with the following content:
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+
+export class LambdaApiStack extends cdk.Stack {
+  constructor(scope: cdk.App, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    // Create the Lambda function
+    const lambdaFunction = new lambda.Function(this, 'SwiftLambdaFunction', {
+      runtime: lambda.Runtime.PROVIDED_AL2,
+      architecture: lambda.Architecture.ARM_64,
+      handler: 'bootstrap',
+      code: lambda.Code.fromAsset('../.build/plugins/AWSLambdaPackager/outputs/AWSLambdaPackager/APIGatewayLambda/APIGatewayLambda.zip'),
+      memorySize: 512,
+      timeout: cdk.Duration.seconds(30),
+      environment: {
+        LOG_LEVEL: 'debug',
+      },
+    });
+ }
+}    
+```
+The code assumes you already built the Swift Lambda function with the `swift package archive --allow-network-connections docker` command. The ZIP file is located in the `.build/plugins/AWSLambdaPackager/outputs/AWSLambdaPackager/MyLambda/MyLambda.zip` folder.
```

Comment:
Again we should be clear about what app we're deploying here so there's no confusion when invoking the lambda etc

Reply:
done, I added a phrase at the start of the section and clarified this one.